featuretools/entityset/entityset.py

```diff
@@ -485,6 +485,11 @@ def entity_from_dataframe(self,
         if time_index is not None and time_index == index:
             raise ValueError("time_index and index cannot be the same value, %s" % (time_index))
 
+        if time_index is None:
+            for variable, variable_type in variable_types.items():
+                if variable_type == vtypes.DatetimeTimeIndex:
+                    raise ValueError("Variable %s must be set using time_index parameter" % (variable))
```

Comment:
Can you also include that the variable is DatetimeTimeIdex type, e.g. DatetimeTimeIndex variable %s.